src/openrct2/object/Object.h

```diff
@@ -26,6 +26,7 @@
 using ObjectEntryIndex = uint16_t;
 constexpr const ObjectEntryIndex OBJECT_ENTRY_INDEX_NULL = std::numeric_limits<ObjectEntryIndex>::max();
 struct ObjectRepositoryItem;
+using ride_type_t = uint16_t;
```

Comment:
I thought we had that define already 🤔

Reply:
Yes, but it’s defined in `RideTypes.h`, which in turn has a dependency on this file. So I cannot include RideTypes.h because it would mean a circular dependency. :/